config.yaml

```diff
@@ -19,6 +19,16 @@ directories:
                 - /mnt/tmp/02
                 - /mnt/tmp/03
 
+       # Optional: Allows overriding some characteristics of certain tmp
+       # directories. This contains a map of tmp directory names to
+       # attributes. If a tmp directory and attribute is not listed here,
+       # it uses the default attribute setting from the main configuration.
```

Comment:
Can we add a note here listing the attributes for which overrides are currently supported (right now, just `tmpdir_max_jobs`)?

Reply:
actually i'll just add this in a followup.